Doc/conf.py

```diff
@@ -22,6 +22,10 @@
     import _tkinter
 except ImportError:
     _tkinter = None
+from os import name as system_name
+is_posix = system_name == 'posix'
+is_windows = system_name == 'nt'
+del system_name
```

Comment:
Or IS_POSIX/IS_WINDOWS? I don't think there is really any precedence for the naming convention here. However having the ability to skip tests depending on platform, gives one the ability to actually have sphinx doctest the code in the documentation rather than just skipping it with :: and trusting that it is correct (as is the current practice throughout much of pathlib's documentation).

Reply:
It doesnt really matter in this snippet that’s invisible to readers of the doc 🙂 

I think lower case looks fine.